docs/content/storage-drivers/azure.md

```diff
@@ -16,6 +16,9 @@ An implementation of the `storagedriver.StorageDriver` interface which uses [Mic
 | `realm`                            | no       | Domain name suffix for the Storage Service API endpoint. For example realm for "Azure in China" would be `core.chinacloudapi.cn` and realm for "Azure Government" would be `core.usgovcloudapi.net`. By default, this is `core.windows.net`.                        |
 | `copy_status_poll_max_retry`       | no       | Max retry number for polling of copy operation status. Retries use a simple backoff algorithm where each retry number is multiplied by `copy_status_poll_delay`, and this number is used as the delay. Set to -1 to disable retries and abort if the copy does not complete immediately. Defaults to 5.                |
 | `copy_status_poll_delay`            | no       | Time to wait between retries for polling of copy operation status. This time is multiplied by N on each retry, where N is the retry number. Defaults to 100ms |
+| `disablekeepalives` | no | Disables HTTP keep-alives on the HTTP tranport when `true`. Each connection to the server will be used for a single HTTP request. The default is `false`. |
+| `disableidleconnections` | no | Disables idle HTTP connections on the HTTP transport when `true`. The default is `false`. |
```

Comment:
They are separate options on the golang transport, which is why I separated them here. We disable both, because we've found better performance in our stack by not allowing those connections to stay open.

Reply:
The only meaningful difference between setting `DisableKeepAlives = true` and `MaxIdleConnsPerHost = -1` is that the former [causes all requests to be made with the request header `Connection: close`.](https://cs.opensource.google/go/go/+/release-branch.go1.21:src/net/http/transport.go;l=2611-2615;bpv=0;bpt=0) Otherwise [they are equivalent.](https://cs.opensource.google/go/go/+/release-branch.go1.21:src/net/http/transport.go;l=925-928;bpv=0;bpt=0) They are separate  options on the `http.Transport` so a nonzero number of idle connections could be configured. Setting `MaxIdleConnsPerHost = -1` is redundant when `DisableKeepAlives = true`, therefore the storage drivers only need a single `disablekeepalives` option.